shared-kcp/add_workload.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+KUBECONFIG="${KUBECONFIG:-$HOME/.kube/config}"
+SCRIPT_DIR="$(cd "$(dirname "$0")" >/dev/null ; pwd)"
+KUBECONFIG_KCP="$SCRIPT_DIR/sharedkcp.kubeconfig"
```

Comment:
Use the same mechanism as KUBECONFIG to be able to override the value from the CLI (or add an optional  parameter).

Reply:
Done.